### DISCUSSION TITLE
hello eyeryone,I would like to ask if there is a demo scene in sofa where two flexible models collide?

### DISCUSSION THREAD

**Discussion by yangwxb** [sofa-framework/sofa#4728] [Write / Run a simulation] [answered]
hello eyeryone,I would like to ask if there is a demo scene in sofa where two flexible models collide?

**Comment by epernod** [comment, marked as answer]
Hi, 
yes you have at least [ChainHybrid](https://github.com/sofa-framework/sofa/blob/master/examples/Demos/chainHybrid.scn) and [CollisionMultiple](https://github.com/sofa-framework/sofa/blob/master/examples/Demos/collisionMultiple.scn)
Then if you dig into the examples related to collision you should have more. 
If your models explode, it certainly mean you need to calibrate your collision stiffness compare to the mechanical property you assign to your models.

best regards

**Reply by yangwxb** [reply]
thanks epernod!
